### PR TITLE
Add Swagger-UI for documenting the API #75

### PR DESCRIPTION
**Pull request by jonattfin**
Added the swagger, with all the methods, but the authentication is still not done. I suggest the commit to be merged.

Endpoint: http://localhost:8080/api/explorer

Right now it looks like this:
https://screenshots.firefox.com/TfaaGozCXbwTkYKZ/localhost

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,497 @@
...
+          example: my post
+        CatName:
+          type: string
+          example: description
+        Ordinal:
+          type: string
+          example: location
+      xml:
+        name: Post
+
```
Comment (lrdTnc) [inline swagger.yaml]: missing type

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,497 @@
...
+          type: string
+        ngoname:
+          type: string
+        ngophone:
+          type: string
+        ngoemail:
+          type: string
+        ngofacebook:
+          type: string
+        ngofacebook:
```
Comment (lrdTnc) [inline swagger.yaml]: duplicate ngofacebook

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,497 @@
...
+        password:
+          type: string
+        secret:
+          type: string
+      xml:
+        name: HashData
+    PhotoData:
+      type: object
+      required:
+      properties:
```
Comment (lrdTnc) [inline swagger.yaml]: missed required and properties? I get an error in swagger editor.

Comment (jonattfin) [reply swagger.yaml]: You are right with all the mentioned comments, i will fix it asap. thx

**Comment by jonattfin** [comment]
I'll make another PR.

**Pull request closed without merging**